rsmtool/utils/notebook.py

```diff
@@ -16,7 +17,16 @@
 
 from IPython.display import HTML, display
 
-HTML_STRING = """<li><b>{}</b>: <a href="{}" download>{}</a></li>"""
+from .constants import INTERMEDIATE_FILES_TO_DESCRIPTIONS
+
+INTERMEDIATE_TABLE_ROW_STRING = """
+<tr>
+    <td style="text-align:left;">
+        <a href="{}" download>{}</a>
```

Comment:
```suggestion
        <a href="{}" style="text-decoration: none" download>{}</a>
```
Suggesting this to avoid underlined links with do not look so good when file names contain underscore.